### PR TITLE
Doc changelog 20221102

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,231 @@
...
+    Add a ``banner`` property representing the banner image for the account.
+
+  ``inet:web:mesg``
+    Add a ``deleted`` property to mark if a message was deleted.
+
+  ``inet:web:post:link``
+    Add a form to record a link contained in the post text.
+
+  ``it:mitre:attack:group``
+    Add a ``isnow`` property to record the potential for MITRE groups to be
```
Comment (Cisphyx) [inline CHANGELOG.rst:39]: 'an' isnow

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,6 +4,231 @@
...
+    deprecated and renamed.
+
+  ``it:mitre:attack:software``
+    Add a ``isnow`` property to record the potential for MITRE software to be
```
Comment (Cisphyx) [inline CHANGELOG.rst:43]: 'an' isnow

File: CHANGELOG.rst [hunk 3]
```diff
@@ -4,6 +4,231 @@
...
+
+  ``media:news``
+    Add a ``topics`` array property to record a list of relevant topics in the
+    article.
+
+  ``media:topic``
+    Add a form for recording different media topics.
+
+  ``meta:rule``
+    Add a ``url`` property to record a url that doucments as rule.
```
[the remark this answers is not in the document]
Comment (Cisphyx) [reply CHANGELOG.rst:66]: also the second URL should be uppercase

File: CHANGELOG.rst [hunk 4]
```diff
@@ -4,6 +4,231 @@
...
+
+    Add a ``ext:id`` property to record a external identifier for the rule.
```
Comment (Cisphyx) [inline CHANGELOG.rst:68]: 'an' ext:id

Comment (Cisphyx) [reply CHANGELOG.rst:68]: 'an' external

File: CHANGELOG.rst [hunk 5]
```diff
@@ -4,6 +4,231 @@
...
+
+  ``meta:sophistication``
+    Add a form to record sophistication score with named values: ``very low``,
+    ``low``, ``medium``, ``high``, and ``very high``.
+
+  ``ou:campaign``
+    Add a ``sophistication`` property to record the assessed sophistication of
+    an campaign.
```
Comment (Cisphyx) [inline CHANGELOG.rst:76]: 'a' campaign

File: CHANGELOG.rst [hunk 6]
```diff
@@ -4,6 +4,231 @@
...
+    Deprecate the ``ou:hasgoal`` form in favor of using the ``ou:org:goals``
+    property.
+
+  ``ou:org``
+    Deprecate the ``techniques`` property in favor of using the ``uses`` light
+    edge.
+
+  ``ou:technique``
+    Add a ``sophistication`` property to record the assessed sophistication of
+    an technique.
```
Comment (Cisphyx) [inline CHANGELOG.rst:91]: 'a' technique

File: CHANGELOG.rst [hunk 7]
```diff
@@ -4,6 +4,231 @@
...
+
+  ``risk:alert``
+    Add a ``url`` property for a URL that documents the alert.
+
+    Add a ``ext:id`` property to record a external ID for the alert.
```
Comment (Cisphyx) [inline CHANGELOG.rst:96]: 'an' ext:id and 'an' external

File: CHANGELOG.rst [hunk 8]
```diff
@@ -4,6 +4,231 @@
...
+  ``risk:attack``
+    Add a ``sophistication`` property to record the assessed sophistication of
+    an attack.
+
+  ``risk:availability``
+    Add a taxonomy for availability status values.
+
+  ``risk:threat``
+    Add a ``sophistication`` property to record the assessed sophistication of
+    an threat cluster.
```
Comment (Cisphyx) [inline CHANGELOG.rst:107]: 'a' threat cluster

File: CHANGELOG.rst [hunk 9]
```diff
@@ -4,6 +4,231 @@
...
+
+    Deprecate the ``techniques`` property in favor of the ``uses`` light edge.
+
+  ``risk:tool:software``
+    Add a ``availability`` property to record the assessed availability of the
```
Comment (Cisphyx) [inline CHANGELOG.rst:112]: 'an' availability

File: CHANGELOG.rst [hunk 10]
```diff
@@ -4,6 +4,231 @@
...
+    tool
```
Comment (Cisphyx) [inline CHANGELOG.rst:113]: missing period after tool

File: CHANGELOG.rst [hunk 11]
```diff
@@ -4,6 +4,231 @@
...
+
+    Migrate the ``type`` property to ``risk:tool:software:taxonomy``.
+
+    Deprecate the ``techniques`` property in favor of the ``uses`` light edge.
+
+  ``risk:tool:software:taxonomy``
+    Rename the type ``risk:tool:taxonomy`` to ``risk:tool:software:taxonomy``.
+
+  ``risk:vuln``
+    Add a ``mitigated`` property to record if a mitigation or fix is avilable
```
Comment (Cisphyx) [inline CHANGELOG.rst:126]: 'avilable'

File: CHANGELOG.rst [hunk 12]
```diff
@@ -4,6 +4,231 @@
...
+    for the vulnerability.
+
+    Add a ``exploited`` property to record if the vulnerability has been
```
Comment (Cisphyx) [inline CHANGELOG.rst:129]: 'an' exploited

File: CHANGELOG.rst [hunk 13]
```diff
@@ -4,6 +4,231 @@
...
+    Add ```nist:nvd:source`` to record the name of the organization which
+    reported the vulnerability in the NVD.
+
+    Add ``nist:nvd:published`` and ``nist:nvd:modified`` to record when the
+    vulnerability was first published, and later modified, in the NVD.
+
+    Add ``cisa:kev:name``, ``cisa:kev:desc``, ``cisa:kev:action``,
+    ``cisa:kev:vendor``, ``cisa:kev:product``, ``cisa:kev:added``,
+    ``cisa:kev:duedate`` properties to record information about the CISA KEV
+    database for the vulnerability.
```
Comment (Cisphyx) [inline CHANGELOG.rst:149]: database 'entry' maybe?

File: CHANGELOG.rst [hunk 14]
```diff
@@ -4,6 +4,231 @@
...
+
+    When used with ``risk:attack`` and ``risk:vuln`` nodes, the edge
+    indicates the attack used the vulnerability.
+
+    When used with ``risk:threat`` and ``risk:vuln`` nodes, the edge
+    indicates the threat cluster uses the vulnerability.
+
+    When used with ``risk:tool:software`` and ``risk:vuln`` nodes, the edge
+    indicates the tool used the vulnerability.
+
```
Comment (Cisphyx) [inline CHANGELOG.rst:182]: i think you're missing a heading for 'targets' here

File: CHANGELOG.rst [hunk 15]
```diff
@@ -4,6 +4,231 @@
...
+    When used with ``risk:attack`` and ``ou:industry`` nodes, the edge
+    indicates the attack targeted the industry.
+
+    When used with ``risk:threat`` and ``ou:industry`` nodes, the edge
+    indicates the threat cluster targets the industry.
+
+    When used with ``risk:tool:software``, the edge indicates the target
```
Comment (Cisphyx) [inline CHANGELOG.rst:189]: should be in the 'used' section, and need a generic one for targets

File: CHANGELOG.rst [hunk 16]
```diff
@@ -4,6 +4,231 @@
...
+    When used with a ``risk:compromise`` node, the edge indicates the target
+    node was stolen or copied as a result of the compromise.
+
+- Add ``$lib.gen.vulnByCve()`` to help generate ``risk:vuln`` nodes for CVEs.
+  (`#2903 <https://github.com/vertexproject/synapse/pull/2903>`_)
+- Add a unary negation operator to Storm expression syntax.
+  (`#2886 <https://github.com/vertexproject/synapse/pull/2886>`_)
+- Add ``$lib.crypto.hmac.digest()`` to compute RFC2104 digests in Storm.
+  (`#2902 <https://github.com/vertexproject/synapse/pull/2902>`_)
+- Update the Storm ``storm:http:resp`` ``.json()`` method to add optional
```
Comment (Cisphyx) [inline CHANGELOG.rst:202]: this entry has a leading period ".json()" vs the next one which is just "decode()", probably remove the period

File: CHANGELOG.rst [hunk 17]
```diff
@@ -4,6 +4,231 @@
...
+  ``encoding`` and ``errors`` arguments, to control how data is deserialized.
+  (`#2898 <https://github.com/vertexproject/synapse/pull/2898>`_)
+- Update the Storm ``bytes``  ``decode()`` method to add an optional
```
Comment (Cisphyx) [inline CHANGELOG.rst:205]: there's an extra space between bytes and decode

File: CHANGELOG.rst (hunk 14 above)
Comment (vEpiphyte) [reply CHANGELOG.rst:182]: That was a bit confusing; let me update that.

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**